Fixed issue with new notification read/archived inbox

LunaLycan287 opened the pull request:
A new notification handling has been introduced where style for unread was marked !important and -read & -archived where added with var(--color-notifications-row-read-bg) !important.
If --bg4 is the correct color can be discussed, I personally like it that way:
- bg5 is read
- bg3 is background of the site and should not be used imho
- bg2 would also be an option maybe
- bg1 is same color as header and should not be used imho

LunaLycan287 committed:
Fixed issue with new notification read/archived inbox

LunaLycan287 commented:
Also I was unsure on how to handle this new case, previously the notifications were handled as I did in my pull request, but also the readme said that "If GitHub uses a var (ie. --color-bg-primary), just redeclare it in :root".

If I should change it to --color-notifications-row-read-bg being declared in :root please tell me.
In the case I should declare it in root, should I also do the same for --color-notifications-row-bg and clean up the existing changes for .notifications-list-item.notification-unread?

CharlieEtienne commented:
Hi @Moonwolf287, thanks for this PR!

> Also I was unsure on how to handle this new case, previously the notifications were handled as I did in my pull request, but also the readme said that "If GitHub uses a var (ie. --color-bg-primary), just redeclare it in :root".

Actually GitHub is starting using CSS vars so I'm replacing bit by bit old declarations like `.notifications-list-item.notification-unread {background-color: var(--bg5); }` with this new system ("redeclare  it in :root")

So, yes, I prefer you to redeclare it in :root like `--color-notifications-row-read-bg: var(--bg4);` for example.

> In the case I should declare it in root, should I also do the same for --color-notifications-row-bg and clean up the existing changes for .notifications-list-item.notification-unread?

Yes, please!

Thank you for your contribution.

Charlie

LunaLycan287 committed:
Changed notification system to use git vars

- moved unread, read & archived color to root
- removed navigation background since it is set by bg-white

LunaLycan287 commented:
I moved the colors for the previously mentioned ones to the root and removed the one for navigation background since this is handled by bg-white now.

I did not touch any of the other notification colors, they are handled by general variables by github like --color-bg-info which already have other values assigned, than the ones currently used by the override and they would clash if I removed the overwrite.

Ready for review and merge

CharlieEtienne committed:
Update version number
